apps/judicial-system/web/src/routes/Court/InvestigationCase/Confirmation/Confirmation.tsx

```diff
@@ -64,23 +63,16 @@ const Confirmation: FC = () => {
   )
   const [modalVisible, setModalVisible] = useState<VisibleModal>('none')
 
-  const isCorrectingRuling = workingCase.notifications?.some(
-    (notification) => notification.type === NotificationType.RULING,
-  )
+  const isCompletedWithoutRuling = Boolean(workingCase.isCompletedWithoutRuling)
```

Comment:
Nit: We can skip the `Boolean` here since `isCompletedWithoutRuling` is an (optional) boolean already and casting this to `Boolean` has no impact on the check below since we will get the same result